app/views/projects/_form.html.erb

```diff
@@ -108,6 +108,20 @@
       <% end %>
     <% end %>
 
+    <% if @readonly %>
+      <div class="col-md-6 mT-15">
+        <% if @project.image.attached?%>
+          <%= f.label :image, "Imagen de proyecto: " %>
+          <%= image_tag(@project.image, {:class => "card-img-top", :alt => "Imagen de proyecto"}) %>
+        <% end %>
+      </div>
+    <% else %>
+      <div class="col-md-6 mT-15">
+        <%= f.label :image, "Imagen de proyecto: " %>
+        <%= f.file_field :image, class: 'form-control' %>
```

Comment:
Remplazar input de archivos genérico por input de bootstrap: (tambien cambiarlo en archivos de gastos y bitácoras)

<div class="input-group">
  <div class="custom-file">
    <input type="file" class="custom-file-input" id="inputGroupFile" aria-describedby="inputGroupFileAddon">
    <label class="custom-file-label" for="inputGroupFile">Escoger Archivo</label>
  </div>
  <div class="input-group-append">
    <button class="btn btn-outline-secondary" type="button" id="inputGroupFileAddon04">
        Cámara
        <i class="ti-camera" aria-hidden="true"></i>
    </button>
  </div>
</div>

Más información en:
https://getbootstrap.com/docs/4.1/components/input-group/#custom-file-input

Reply:
La linea 123, del input, sería con los tabs de ruby, como está en la 121

Reply:
bfdb3414b932ce79a4d8f1b2f42c0e1fb9886cf9